src/Rules/Keywords/RequireFileExistsRule.php

```diff
@@ -0,0 +1,78 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Rules\Keywords;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\Include_;
+use PHPStan\Analyser\Scope;
+use PHPStan\Rules\Rule;
+use PHPStan\Rules\RuleError;
+use PHPStan\Rules\RuleErrorBuilder;
+use PHPStan\ShouldNotHappenException;
+use function is_file;
+use function is_string;
+use function sprintf;
+
+/**
+ * @implements Rule<Include_>
+ */
+final class RequireFileExistsRule implements Rule
+{
+
+	public function getNodeType(): string
+	{
+		return Include_::class;
+	}
+
+	public function processNode(Node $node, Scope $scope): array
+	{
+		if ($this->shouldProcessNode($node)) {
+			$filePath = $this->resolveFilePath($node, $scope);
+			if (is_string($filePath) && !is_file($filePath)) {
+				return [
+					$this->getErrorMessage($node, $filePath),
+				];
+			}
+		}
+
+		return [];
+	}
+
+	private function shouldProcessNode(Node $node): bool
+	{
+		return $node instanceof Include_ && (
+				$node->type === Include_::TYPE_REQUIRE
+				|| $node->type === Include_::TYPE_REQUIRE_ONCE
```

Comment:
What about include()? I want to report that too.

Reply:
Fixed in https://github.com/phpstan/phpstan-src/pull/3294/commits/10c9ff79ec679a343d292231744bb6f7a905e974